x-pack/test/apm_api_integration/tests/alerts/helpers/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const APM_ALERTS_INDEX = '.alerts-observability.apm.alerts-*';
+export const APM_RULE_CONNECTOR_INDEX = 'apm-rule-connector-index-name';
+export const KIBANA_EVENT_LOG_INDEX = '.kibana-event-log-*';
+export const RULE_ENDPOINT = '/api/alerting/rule';
+export const CONNECTOR_ENDPOINT = '/api/actions/connector';
```

Comment:
Most of these are only used inside the  `alert_test_helper.ts`. It'll be easier to read if they are defined there.